pkg/deploy/api/test/ok.go

```diff
@@ -160,6 +160,36 @@ func OkPodTemplate() *kapi.PodTemplateSpec {
 	}
 }
 
+func OkPodTemplateMissingImageWithOkImageChangeTrigger() *kapi.PodTemplateSpec {
```

Comment:
it's just OkPodTemplateMissingImage right?  Whether it gets used w/ an OkImageChangeTrigger or not is up to the test (but of course it should fail validation if it's not used with an ok ICT)